src/components/game/Game.js

```diff
@@ -295,7 +293,7 @@ class Game extends React.Component {
             return <GameOverview
                 gameModel={this.state.gameModel}
                 users={this.state.users}
-                skipped={this.state.skipped}
+                roundsPlayed={this.state.gameModel.round - 1}
```

Comment:
Are you sure that this is working correctly? Is the round of the gameModel not the actual number?

What is now the different between giving the number of rounds with the props or the actual number. It is still giving information through different components...........

Reply:
It doesn't track the skipped words in the frontend anymore, and skipped is removed.